README.md

```diff
@@ -232,23 +245,13 @@ The best way to get compatible dependencies on Windows is to use
 ```
 D:\> git clone https://github.com/Microsoft/vcpkg.git
 D:\> cd vcpkg
-D:\> .\bootstrap-vcpkg.bat
-D:\vcpkg> .\vcpkg integrate install
+D:\vcpkg> .\bootstrap-vcpkg.bat -disableMetrics
 ...
 CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=D:/vcpkg/scripts/buildsystems/vcpkg.cmake"
 ```
 
-Then install the libraries. For a 64-bit build, run:
-
-```
-D:\vcpkg> .\vcpkg install libpng:x64-windows libjpeg-turbo:x64-windows llvm[target-all,clang-tools-extra]:x64-windows
-```
-
-To support 32-bit builds, also run:
-
-```
-D:\vcpkg> .\vcpkg install libpng:x86-windows libjpeg-turbo:x86-windows llvm[target-all,clang-tools-extra]:x86-windows
-```
+When using the toolchain file, vcpkg will automatically build all the necessary
+dependencies.
```

Comment:
However, as stated above, be aware that acquiring LLVM in this way may use over 100 GB of disk space for its build trees and take a very long time to build. You can manually delete the build trees afterwards, but vcpkg will not do this automatically.

Reply:
Added the warning 👍